### PR TITLE
I added email to sign up form, looks like it is working, but I will n…

### PR DESCRIPTION
**Pull request by olavea**
…eed to check #101

### CODE REVIEW

File: src/components/UserForm.js [hunk 1]
```diff
@@ -124,6 +125,20 @@ const UserForm = ({ variant, standalone = true, onSubmitFulfilled }) => {
           onChange={handleChange("username")}
           InputLabelProps={{ shrink: true }}
         />
+        <TextField
```
Comment (raae) [inline src/components/UserForm.js]: It should only be displayed when variant === "signup"

Comment (olavea) [reply src/components/UserForm.js]: @olavea
I removed email from log in form, looks like it is working,
😸 👍

Comment (olavea) [reply src/components/UserForm.js]:
```
       {variant === "signup" ? (
          <TextField
            variant="outlined"
            margin="normal"
            required
            fullWidth
            id="email"
            label="Email"
            name="email"
            placeholder="unicorn@usepow.app"
            value={state.email}
            autoComplete="email"
            onChange={handleChange("email")}
            InputLabelProps={{ shrink: true }}
          />
        ) : (
          "current-email"
        )}
 
```

Comment (olavea) [reply src/components/UserForm.js]: I learned a new thing:

If a customer 🎩 is signing UP ⬆️ she can write her email:  

```
{variant === "signup" ? (
  <TextField
  ...
    label="Email"
  ...
    />
  ) : (
    "current-email"
)}
```

Else a customer 🎩 can not write her email.

#usepowapp #100daysofGatsby #webdevnewbie

File: src/components/UserForm.js [hunk 2]
```diff
@@ -151,7 +155,7 @@ const UserForm = ({ variant, standalone = true, onSubmitFulfilled }) => {
           autoComplete={
             variant === "signup" ? "new-password" : "current-password"
           }
-          placeholder="glitter-rainbow-butterfly-kitty"
+          placeholder="glittery-kitty-rainbowy-butterfly"
```
Comment (raae) [inline src/components/UserForm.js]: I want the old one back.

File: src/components/UserForm.js [hunk 3]
```diff
@@ -125,20 +125,24 @@ const UserForm = ({ variant, standalone = true, onSubmitFulfilled }) => {
...
+            label="Email"
+            name="email"
+            placeholder="unicorn@usepow.app"
+            value={state.email}
+            autoComplete="email"
+            onChange={handleChange("email")}
+            InputLabelProps={{ shrink: true }}
+          />
+        ) : (
+          "current-email"
```
Comment (raae) [inline src/components/UserForm.js]: Nothing should happen if login. Try using `variant === "signup" && <Text...`

**Comment by olavea** [comment]
Do you want the email to show up in the stripe checkout like this:

POW-signup-email_122

**Comment by raae** [comment]
Yes I do, but that might not be possible without some more serverless functions.

We might have to:

1. Make a stripe customer using a serverless function
2. Then create a payment checkout session with reference to the customer returned in 1
3. Then use `stripe.redirectToCheckout` with the session id from 2, instead of off all the info, we send now.

**Comment by raae** [comment]
To summarize, I can accept this PR when you fix the two notes I had. We will just let people type their email twice.

File: src/components/UserForm.js (hunk 3 above)
Comment (olavea) [reply src/components/UserForm.js]: I did what you said and 
POW!👊
Works!






POW-POW!👊_Works!-126

**Comment by olavea** [comment]
I fixed the two notes you had 
👍

File: src/components/UserForm.js (hunk 2 above)
Comment (olavea) [reply src/components/UserForm.js]: Se commit below.
Placeholder is now "glitter-rainbow-butterfly-kitty"

👍

**Comment by raae** [comment]
I merged did the merge in master, via the button here on Github.

You should now be able to squash and merge.

**Pull request merged**